tlp-pcilist

```diff
@@ -58,9 +62,9 @@ foreach (`lspci -m`) {
         # get Runtime PM mode
         my $pmode = catsysf ("$devc");
         # output pm mode and device data
-        printf "%s/power/control = %-4s (%s, %s, %s)\n", $devp, $pmode, $class, $classdesc, $driver;
+        printf "%s/power/control = %-4s  %s/power/runtime_status = %-4s (%s, %s, %s)\n", $devp, $pmode, $devp, $runtime_status, $class, $classdesc, $driver;
     } else { # control file missing --> output device data only
-        printf "%s/power/control = (not available) (%s, %s, %s)\n", $devp, $class, $classdesc, $driver;
+        printf "%s/power/control = (not available) %s/power/runtime_status = %-4s (%s, %s, %s)\n", $devp, $devp, $runtime_status, $class, $classdesc, $driver;
```

Comment:
I guess if `control` is not there, then `runtime_status` will not be either. 
Suggestion: do not show it here.